wrappers/wrapper.rb

```diff
@@ -931,23 +931,21 @@ def unassigned_rests(vrp)
     end
 
     def expand_vehicles_for_consistent_empty_result(vrp)
-      if vrp.scheduling? && !vrp.schedule_expanded_vehicles
-        periodic = Interpreters::PeriodicVisits.new(vrp)
-        periodic.generate_vehicles(vrp)
-      else
-        vrp.vehicles
-      end
+      periodic = Interpreters::PeriodicVisits.new(vrp)
+      periodic.generate_vehicles(vrp)
     end
 
-    def empty_result(solver, vrp, unassigned_reason = nil)
-      vrp.vehicles = expand_vehicles_for_consistent_empty_result(vrp)
+    def empty_result(solver, vrp, unassigned_reason = nil, already_expanded = true)
```

Comment:
we could progressively migrate from multiple optional parameters, to a single hash `options` which contains those keys and values ?

Reply:
I tried this, but we sometimes need to overload only one of those. 

Wouldn't this force us to provide them all ? For instance if we provide unassigned_reason then we need to specify already_expanded, right ?

Reply:
I think its better if we can keep this PR as simple as possible. I need it to rebase my PR which is a prerequisite to another PR which I am trying to create...

Lets create a ticket for this "options as hash" idea and discuss the details on the ticket but I think I am already sold :) 

@fonsecadeline it doesn't force us https://codereview.stackexchange.com/a/73503. Instead of variables, they become hash keys, and this forces the caller to write the name of the key.

Like this for example, `empty_result(solver, vrp, already_expanded: true)` here there is a "hidden" hash in fact `empty_result(solver, vrp, { already_expanded: true })` of course we need to decide for a way to separate "real" arguments from "options"